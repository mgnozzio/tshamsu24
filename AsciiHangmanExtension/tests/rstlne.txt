rstlne
r
s
t
l
n
e
