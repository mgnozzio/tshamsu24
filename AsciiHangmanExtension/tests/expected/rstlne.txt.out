Welcome to the ASCII Version of Hangman!
For best results, please run this from the command line.
Please enter a secret word: 
















































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































   ____  
  |    | 
  |      
  |      
  |      
__|__    
Puzzle to solve: _ _ _ _ _ _ 
Please guess a letter: R
Puzzle to solve: R _ _ _ _ _ 
Please guess a letter: S
Puzzle to solve: R S _ _ _ _ 
Please guess a letter: T
Puzzle to solve: R S T _ _ _ 
Please guess a letter: L
Puzzle to solve: R S T L _ _ 
Please guess a letter: N
Puzzle to solve: R S T L N _ 
Please guess a letter: E
Success!  Player 2 wins!
